docs/ruleset_functions/get_heating_fuels_p.md

```diff
@@ -0,0 +1,75 @@
+# get_heating_fuels_p
+
+**Description:** Get a list of the heating fuels used in the P-RMR.  
+
+**Inputs:**
+- **P-RMR**: To determine the heating fuels used in the P-RMR.
+
+**Returns:**
+- **heating_fuels_list_p**: A list that saves all the heating fuels used in the P_RMR.
+ 
+**Function Call:** None
+
+
+## Logic:  
+Set boolean variables to FALSE. Default is fuel is not used for space heating in the P_RMR. These boolean variable will be set to true as this function goes through the heating sources in the P_RMR and determines if each specific fuel has been modeled.
+- Set boolean variable to FALSE: `fuels_propane_p = FALSE`
+- Set boolean variable to FALSE: `fuels_natural_gas_p = FALSE`
+- Set boolean variable to FALSE: `fuels_electricity_p = FALSE`
+- Set boolean variable to FALSE: `fuels_oil_p = FALSE`
+- Set boolean variable to FALSE: `fuels_other_p = FALSE`
+
+Loop through boilers to determine the fuels used by the boilers in the P_RMR. Boolean variables are set to TRUE accordingly.
+- For each rulesetModelInstance_p in the P_RMR: `for rulesetModelInstance_p in P_RMR:`
+    - For each boiler_p in rulesetModelInstance_p.boilers: `for boiler_p in rulesetModelInstance_p.boilers:`
```

Comment:
Some boilers may be serving process loads. Do we want to include them also? What about fuels used for ServiceWaterHeatingEquipment.heater_fuel_type? If we are only interested in space heating, then #24 would need to be expanded.

Reply:
maybe we start with the function which gets all HVAC systems in a building based on all zones in a building and then identify all the heating fuel types for all HVAC systems.

Reply:
I think we need to include all boilers whether serving process or SWH because if natural gas issued for anything in the proposed then it is likely available at the building. Or maybe this is not a safe assumption?